Data/Help/1/script.html

```diff
@@ -401,6 +401,8 @@ <h3><u>Character Script Commands</u></h3>
         <li>See "<a href="#branchingexamples">Examples of branching code
              structures</a>" below.</li>
       </ol></li>
+  <li><a name="while"><b>While ...</b></a> - This command functions like <a href="#if"><b>If ...</b></a>, and is paired with a <a href="#whileend"><b>While End</b></a> to create a loop.</li>
```

Comment:
Can you expand the explanation for those who are not programmers? I think maybe something like this would be a bit more beginner friendly:

> "While" and "While End" function exactly like "If/If End" with one exception - when "While End" is reached, the script returns back to its matching "While" command, effectively rerunning the inner script block repeatedly until the condition is false.

Or just take pieces of it.

Reply:
Yeah, this is a better description. I'll have a bit of a think about it, but I'll probably go with this.